lib/cartopy/mpl/feature_artist.py

```diff
@@ -186,12 +185,10 @@ def draw(self, renderer, *args, **kwargs):
             # in view.
             geoms = self._feature.intersecting_geometries(extent)
 
-        # Freeze the kwargs so that we can use them as a dict key. We will
-        # need to unfreeze this with dict(frozen) before passing to mpl.
-        prepared_kwargs = _freeze(kwargs)
-
         # Project (if necessary) and convert geometries to matplotlib paths.
-        stylised_paths = OrderedDict()
+        stylised_paths = OrderedDict()  # used with styler
+        unstyled_paths = []             # used if there is no styler
```

Comment:
Do we need OrderedDict anymore? The versions of Python we support are now insertion order guaranteed.